sql-odbc/build_mac_release64.sh

```diff
@@ -2,13 +2,21 @@
 # $BITNESS=64
 
 cd src
-git clone -b "1.7.329" "https://github.com/aws/aws-sdk-cpp.git"
+git clone -b "2022.09.27" "https://github.com/microsoft/vcpkg.git"
```

Comment:
I think it can use vcpkg that comes with macos runner. See [here](https://github.com/actions/runner-images/blob/main/images/macos/macos-12-Readme.md) for exact version.